com.zenorogue.HyperRogue.json

```
@@ -0,0 +1,58 @@
+{
+  "app-id":"com.zenorogue.HyperRogue",
```

```suggestion
  "app-id":"com.roguetemple.HyperRogue",
```

Based http://www.roguetemple.com/z/hyper/ (I followed from the github repo)

this would mean renaming the files etc.